app/src/main/java/com/github/jorgecastillo/kotlinandroid/KotlinArchitectureApp.kt

```diff
@@ -1,16 +1,27 @@
 package com.github.jorgecastillo.kotlinandroid
 
 import android.app.Application
-import kategory.Id
+import com.github.jorgecastillo.architecturecomponentssample.model.error.CharacterError
+import com.github.jorgecastillo.kotlinandroid.di.context.GetHeroesContext
+import com.github.jorgecastillo.kotlinandroid.functional.AsyncResult
+import com.github.jorgecastillo.kotlinandroid.functional.MonadControl
+import kategory.GlobalInstances
+import kategory.InstanceParametrizedType
 
 class KotlinArchitectureApp : Application() {
 
-  override fun onCreate() {
-    super.onCreate()
-    initGlobalInstances()
-  }
+    override fun onCreate() {
+        super.onCreate()
```

Comment:
In all projects I have been part of formatting is enforced on successful compilation. For example using `scalafmt`. Do you mean in Android everyone has to manually configure their IDE to get the same formatting?